### ISSUE TITLE
Testing Play application leads to error

### ISSUE THREAD

**Issue by ChristophEwertowski** [hbz/digitalisiertedrucke#54]
If I run ```~/activator-1.3.10-minimal/bin/activator test``` in the folder git/digitalisiertedrucke I get the following error: 
```
[error] Test IntegrationTest.test failed: com.google.inject.ProvisionException: Unable to provision, see the following errors:
[error] 
[error] 1) Error injecting constructor, BindHttpException[Failed to bind to [6011]]; nested: ChannelException[Failed to bind to: localhost/127.0.0.1:6011]; nested: BindException[Die Adresse wird bereits verwendet]; 
```

I asked fsteeg how to find out which application uses that port but ```ps aux | grep 6011``` didn't lead to any satisfactory result.

**Comment by dr0i** [comment]
`ps aux` will just list all processes, there names , the owner, the id and the parent id, the starting time ... but *not* the port on which the process is running.
To ensure to list all programs which use that port (be it as host or client) do as root:
`# lsof -i:6011`

**Comment by ChristophEwertowski** [comment]
Accidentally deleted the branch with the error. When I set it up new at a folder called "git2" it couldn't built because the index wasn't created and then the integrationTest failed. Now that I deleted the old branch "git" (because the changes are already in the master branch), renamed the newer branch git2 into git and let the test of the activator run anew, it works again. 
Anyway, closing this issue.
